mitosheet/mitosheet/step_performers/graph/graph.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+from copy import deepcopy
+from typing import Any, Dict, List, Optional, Set, Tuple
+import pandas as pd
+
+from mitosheet.state import State
+from mitosheet.step_performers.graph.graph_utils import GRAPH_TITLE_LABELS, get_html_and_script_from_figure
+from mitosheet.step_performers.graph.plotly_express_graphs import (
+    get_plotly_express_graph,
+    get_plotly_express_graph_code,
+)
+from mitosheet.step_performers.step_performer import StepPerformer
+
+
+class GraphStepPerformer(StepPerformer):
+    """
+    Creates a graph of the passed parameters and update the graph_data_json
+
+    {
+        preprocessing: {
+            safety_filter_turned_on_by_user: boolean
+        },
+        graph_creation: {
```

Comment:
add graph_styling to these events now so that we don't need to do a step upgrade